src/lib.rs

```diff
@@ -1201,13 +1200,14 @@ pub unsafe trait TryFromBytes {
 
     /// Attempts to interpret the prefix of the given `candidate` as a `&Self`
     /// without copying.
-    ///
-    /// If the first `size_of::<Self>()` bytes of `candidate` are a valid
-    /// instance of `Self`, this method returns both a reference to those bytes
-    /// interpreted as a `Self`, and a reference to the remaining bytes. If
-    /// `candidate.len() < size_of::<Self>()` or `candidate` is not aligned to
-    /// `align_of::<Self>()` or the bytes are not a valid instance of `Self`,
-    /// this returns `Err`.
+    /// 
+    /// This method attempts to deserialize the largest possible size of `Self`
```

Comment:
Can we pick a different verb besides "deserialize"? IMO it has connotations of copying.